docs/modules/help/pages/reporting_hours.adoc

```diff
@@ -0,0 +1,19 @@
+= Accessing the Worked Hours Report
+
+WARNING: Depending on your access rights, your actual screen might differ from what is shown in the screenshots in this help system.
+
+[[overview]]
+== Overview
+
+Thanks to the "Report Hours" screen, customers are notified in real time of the time and effort required to solve tickets. This information is updated in real time as VSHNeers are busy solving issues.
```

Comment:
We might reiterate the warning on the page here again: It's not the invoice, hours get controlled by the Service Manager before the invoice gets sent.